roles/StackStorm.rabbitmq/tasks/rabbitmq_redhat.yml

```diff
@@ -0,0 +1,46 @@
+---
+
+- name: Setup rabbitmy/erlang repo from packagecloud for RH
+  become: yes
+  no_log: yes
+  yum_repository:
+    name: rabbitmq_erlang
+    description: rabbitmq_erlang
+    baseurl: https://packagecloud.io/rabbitmq/erlang/el/{{ ansible_facts.distribution_major_version }}/$basearch
+    repo_gpgcheck: yes
+    gpgkey: "https://packagecloud.io/rabbitmq/erlang/gpgkey"
+    sslcacert: /etc/pki/tls/certs/ca-bundle.crt
+    metadata_expire: 300
+    gpgcheck: no
+    enabled: yes
+    sslverify: yes
+  when: ansible_facts.os_family == 'RedHat'
```

Comment:
Considering the file is named `rabbitmq_redhat.yml` which is included conditionally depending on OS,
I think this line could be removed.
```suggestion
```